go.mod

```diff
@@ -1,3 +1,15 @@
 module github.com/GoogleCloudPlatform/opentelemetry-operations-go
 
 go 1.14
+
+require (
+	cloud.google.com/go v0.57.0 // indirect
+	contrib.go.opencensus.io/exporter/stackdriver v0.13.1
```

Comment:
We shouldn't have this dependency, as we want to eventually get rid of dependency on the OpenCensus code.

Reply:
Yes. The Opencensus autodetect would be removed from exporter dependency when we are able to get these monitored resources from metrics exporter. We can finally completely get rid of it when the autodetect feature in OpenTelemetry is implemented.